nio/events/account_data.py

```diff
@@ -67,6 +69,28 @@ def from_dict(cls, event_dict):
         )
 
 
+@attr.s
+class TagEvent(AccountDataEvent):
+    """Tag event.
+
+    Informs the client tags of a room.
```

Comment:
Could you describe a little bit more what room tags are used for? I know that the documentation isn't stellar in the rest of the lib but I'm trying to improve it.